src/appservice-kube/azext_appservice_kube/_constants.py

```diff
@@ -11,8 +11,8 @@
 KUBE_FUNCTION_APP_KIND = 'linux,kubernetes,functionapp'
 KUBE_FUNCTION_CONTAINER_APP_KIND = 'linux,kubernetes,functionapp,container'
 
-LINUX_RUNTIMES = ['dotnet', 'node', 'python', 'java', 'powershell', 'dotnet-isolated']
-WINDOWS_RUNTIMES = ['dotnet', 'node', 'java', 'powershell', 'dotnet-isolated']
+LINUX_RUNTIMES = ['dotnet', 'node', 'python', 'java', 'powershell', 'dotnet-isolated', 'custom']
```

Comment:
Are all these used/ referenced in Kube code?

Reply:
These aren't used directly by `create_functionapp`, but they are used as in the `_params.py` file to determine what the allowed values are for `--runtime`. These hardcoded constants will be removed by this change: https://github.com/Azure/azure-cli-extensions/pull/4384 once the runtime stacks update is released